src/modules/map/components/LocationPopupCopySlot.vue

```diff
@@ -1,48 +1,51 @@
 <template>
-    <!-- eslint-disable vue/no-v-html-->
     <button
+        id="copyButton"
         ref="button"
         class="btn btn-light btn-sm"
         type="button"
         @click="copyValue"
-        v-html="$t(buttonText)"
-    />
-    <!-- eslint-enable vue/no-v-html-->
+    >
+        <FontAwesomeIcon class="icon" :icon="['far', 'copy']" />
+    </button>
 </template>
 
 <script>
 import log from '@/utils/logging'
+import tippy from 'tippy.js'
+import 'tippy.js/dist/tippy.css' // optional for styling
 
 export default {
     props: {
         value: {
             type: String,
             default: '',
         },
-        resetDelay: {
-            type: Number,
-            default: 1000,
-        },
-    },
-    data() {
-        return {
-            buttonText: 'copy_cta',
-        }
     },
-    unmounted() {
-        clearTimeout(this.resetTimeout)
+    mounted() {
+        tippy('#copyButton', {
+            content: this.$i18n.t('copy_cta'),
+            arrow: true,
+            placement: 'right',
+            touch: 'hold',
+        })
```

Comment:
I would maybe save the instances returned by both constructor, and then call https://atomiks.github.io/tippyjs/v6/methods/#destroy in the `unmount` lifecycle hook, so that we do not create unnecessary in-memory instances (could do a little bit of memory leakage there...)